pkg/database/realm.go

```diff
@@ -816,3 +823,34 @@ func (r *Realm) RenderWelcomeMessage() string {
 	raw := blackfriday.Run([]byte(msg))
 	return string(bluemonday.UGCPolicy().SanitizeBytes(raw))
 }
+
+// ToCIDRList converts the newline-separated and/or comma-separated CIDR list
+// into an array of strings.
+func ToCIDRList(s string) []string {
+	var cidrs []string
+	for _, line := range strings.Split(s, "\n") {
+		for _, v := range strings.Split(line, ",") {
+			v = strings.TrimSpace(v)
+
+			// Ignore blanks
+			if v == "" {
+				continue
+			}
+
+			// If there's no /, assume the most specific. This is intentionally
+			// rudimentary.
+			if !strings.Contains(v, "/") {
+				if strings.Contains(v, ":") {
+					v = fmt.Sprintf("%s/128", v)
+				} else {
+					v = fmt.Sprintf("%s/32", v)
+				}
+			}
+
+			cidrs = append(cidrs, v)
```

Comment:
should we ParseCIDR and check for errors here?

Reply:
Done